src/store/modules/swap/ethBancor.ts

```diff
@@ -3717,10 +3718,17 @@ export class EthBancorModule
         this.liquidityProtectionSettings.contract,
         w3
       );
-      const [balances, limit] = await Promise.all([
+      const liquidityProtectionSystemStore = buildLiquidityProtectionSystemStoreContract(
+        "0xc4C5634De585d43DaEC8fA2a6Fb6286cd9B87131",
```

Comment:
We need to source this from on-chain, can't assume this will remain static and won't be the same on ropsten.

Reply:
Yeah as I said In my comment I need to know where to get it from